expat/PSPBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: Wouter Wijsman <wwijsman@live.nl>
+pkgname=expat
+pkgver=2.1.0
+pkgrel=1
+pkgdesc="XML parsing C library"
+arch=('mips')
+url="https://libexpat.github.io/"
+license=('MIT')
+depends=()
+makedepends=()
+optdepends=()
+source=(
+	"http://sourceforge.net/projects/expat/files/${pkgname}/${pkgver}/${pkgname}-${pkgver}.tar.gz"
+	"config.sub"
+)
+sha256sums=(
+	'823705472f816df21c8f6aa026dd162b280806838bb55b3432b0fb1fcca7eb86'
+	'SKIP'
+)
+
+prepare() {
+  cd "$pkgname-$pkgver"
+  cp ../config.sub config.sub
+  cp ../config.sub conftools/config.sub
+}
+
+build() {
+	cd "$pkgname-$pkgver"
+	LDFLAGS="$LDFLAGS -L$(psp-config --pspsdk-path)/lib -L$(psp-config --psp-prefix)/lib -lc -lpspuser" \
+    LIBS="$LIBS -lc -lpspuser" \
+    ./configure --host=psp --prefix=/psp
+	make
+}
+
+package() {
+	cd "$pkgname-$pkgver"
+	make DESTDIR="$pkgdir/" install
+	
+	install -Dm 644 COPYING "$pkgdir/psp/share/licenses/$pkgname/COPYING"
+
+	# Remove the binary we can't use
```

Comment:
Might want to disable with configure:
https://github.com/libexpat/libexpat/blob/master/expat/configure.ac#L128-L144
or build with cmake, needs version update though.

Reply:
Yeah, the version we ship does not have this option.

Reply:
Needs 2.2.4 at least, but might be worth to use latest.

Reply:
True, but I don't really want to work on that for this PR. It's enough work without upgrades